README.md

```diff
@@ -532,6 +532,41 @@ Record & Tuple is built to interoperate with objects and arrays well: you can re
 
 Developers used to manipulating objects in an immutable manner (such as transforming pieces of Redux state) will be able to continue to do the same manipulations they used to do on objects and arrays, this time, with more guarantees.
 
+## Why are Record & Tuple not based on `.get()`/`.set()` methods like [Immutable.js](https://immutable-js.github.io/immutable-js/)?
+
+If we want to keep access to Record & Tuple similar to Objects and Arrays as described in the previous section, we can't rely on methods to perform that access. Doing so would require us to branch code when trying to create a "generic" function able to take Objects/Arrays/Records/Tuples. We also want to avoid an ecoystem split where libraries choose which structures they want to operate on.
```

Comment:
With the `items` proposal, this would no longer be true for Arrays/Tuples.

Reply:
Agreed, that would make it possible for that operation to be generic. It does seem impossible to fix that with objects as adding accessor methods would possibly clash with keys.

Reply:
Which proposal are you referring to? The `item` method?

Even if some APIs to access Tuples vs Arrays are generic, this proposal aims to have greater ergonomics by going for maximal compatibility. I hope this simplifies the mental model for programmers, to minimize the number of differences they have to remember.